.vscode/settings.json

```diff
@@ -46,7 +46,7 @@
   "python.linting.flake8Enabled": true,
   "python.formatting.provider": "black",
   // https://github.com/DonJayamanne/pythonVSCode/issues/992
-  "python.pythonPath": ".venv/bin/python",
+  "python.pythonPath": "C:\\Users\\byk\\AppData\\Local\\Programs\\Python\\Python38\\python.exe",
```

Comment:
This doesn't seem right.

Reply:
Just because you don't use Windows and your username is not `byk` doesn't make it wrong :P

Reply:
Fixed, thanks for the heads up! 😄